### PR TITLE
samba-build: add ability to test rpm install

### PR DESCRIPTION
**Pull request by obnoxxx**
Add means to test rpm install in centos7 and centos8 vms.

### CODE REVIEW

File: ansible/del.vm.centos7.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+- hosts: localhost
+  connection: local
+  become: no
+  vars_files:
+    - vars.yml
```
Comment (anoopcs9) [inline ansible/del.vm.centos7.yml:6]: Do we have _vars.yml_?

File: ansible/roles/test.rpms.centos/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    enabled: yes
+    gpgcheck: no
+
+- name: try installing our samba packages
+  yum:
+    name: "{{ item }}"
+    state: latest
+  loop:
+    - samba
+    - samba-test
```
Comment (anoopcs9) [inline ansible/roles/test.rpms.centos/tasks/main.yml:60]: We may have to try installing _samba-vfs-glusterfs_ as it is not pulled in by _samba_ or _ samba-test_

File: vagrant/Vagrantfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby et ts=2 sts=2 sw=2 :
+
+nodes = [
+  { :hostname => "centos8", :box => "centos/8" },
+  { :hostname => "centos7", :box => "centos/7" },
+]
+
+Vagrant.configure("2") do |config|
```
Comment (anoopcs9) [inline vagrant/Vagrantfile:9]: Don't we need `cpu_mode` configuration to make it run on CentOS CI? Are there any other missing configuration needed for CentOS CI?

File: ansible/del.vm.centos7.yml (hunk 1 above)
Comment (obnoxxx) [reply ansible/del.vm.centos7.yml:6]: https://github.com/gluster/samba-integration/blob/samba-build/ansible/vars.yml

File: vagrant/Vagrantfile (hunk 3 above)
Comment (obnoxxx) [reply vagrant/Vagrantfile:9]: Probably yes, good point. I didn't run in on centos ci yet.

File: ansible/roles/test.rpms.centos/tasks/main.yml (hunk 2 above)
Comment (obnoxxx) [reply ansible/roles/test.rpms.centos/tasks/main.yml:60]: good idea!

File: ansible/del.vm.centos7.yml (hunk 1 above)
Comment (anoopcs9) [reply ansible/del.vm.centos7.yml:6]: > https://github.com/gluster/samba-integration/blob/samba-build/ansible/vars.yml

Ah..my bad.

File: vagrant/Vagrantfile (hunk 3 above)
Comment (obnoxxx) [reply vagrant/Vagrantfile:9]: updated

File: ansible/roles/test.rpms.centos/tasks/main.yml (hunk 2 above)
Comment (obnoxxx) [reply ansible/roles/test.rpms.centos/tasks/main.yml:62]: updated

**Comment by obnoxxx** [comment]
Updated to address the review requests @anoopcs9

**Pull request merged**